pkg/cloudprovider/providers/azure/azure_test.go

```diff
@@ -67,6 +67,104 @@ func TestReconcileLoadBalancerAddPort(t *testing.T) {
 	validateLoadBalancer(t, lb, svc)
 }
 
+// Test additional of a new service/port on an internal LB with a subnet.
+func TestReconcileLoadBalancerAddPortOnInternalSubnet(t *testing.T) {
+	az := getTestCloud()
+	svc := getTestService("servicea", v1.ProtocolTCP, 80)
+	addTestSubnet(&svc)
+	configProperties := getTestPublicFipConfigurationProperties()
```

Comment:
This is logically incorrect, the property is for public IP case, though not validated in reconcileLoadBalancer.